solutions/selection_sort.py

```diff
@@ -0,0 +1,82 @@
+import random
+import time
+
+# O(1) space complexity, O(n^2) time complexity
+def selection_sort_in_place(list):
+
+    for i in range(len(list)):
+
+        min_element = list[i]
+
+        for j in range(i, len(list)):
```

Comment:
Loop can be started from `i+1` instead of `i` since the first comparison would always be with itself if it starts at `i`.